data/pizza.json

```diff
@@ -365,8 +365,8 @@
 				"hidden_ingredients": [
 					"tomatsås", "ost"
 				],
-				"price": 90,
-				"family_price": 200
+				"price": 95,
+				"family_price": 215
```

Comment:
[Should be 210](https://sannegarden.com/meny/johanneberg/#:~:text=20.%20Napoli,95Kr%20210Kr)